### PR TITLE
Update etcdadm controller references to use AWS org

### PR DESCRIPTION
**Pull request by abhay-krishna**
After aws/eks-anywhere-build-tooling#860 merged, the artifacts for the etcdadm repos will reference the AWS org in the URI instead of `mrajashree`. This PR updates the release tooling to fetch the appropriate artifacts from the new S3/ECR locations.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: release/pkg/utils/utils.go [hunk 1]
```diff
@@ -27,6 +27,7 @@ import (
 const hexAlphabet = "0123456789abcdef"
 
 func ExecCommand(cmd *exec.Cmd) (string, error) {
+	fmt.Println(cmd.String())
```
Comment (ewollesen) [inline release/pkg/utils/utils.go:30]: Should this be written via a log statement or behind a debug flag or something? Or did you mean to leave this in just like this?

**Review by ewollesen — COMMENTED** [review]
Just had the one comment/question. I can approve as-is if desired.

File: release/pkg/utils/utils.go (hunk 1 above)
Comment (abhay-krishna) [reply release/pkg/utils/utils.go:30]: Oh my bad this was used for testing. I'll remove it.

**Review by ewollesen — APPROVED** [review]
/lgtm

**Pull request merged**